src/languages/nestedtext.js

```diff
@@ -0,0 +1,85 @@
+/*
+Language: NestedText
+Description: NestedText is a file format for holding data that is to be entered, edited, or viewed by people.
+Website: https://nestedtext.org/
+Category: config
+*/
+
+/** @type LanguageFn */
+export default function(hljs) {
+  const NESTED = {
+    match: [
+      /^\s*(?=\S)/, // have to look forward here to avoid polynomial backtracking
+      /[^:]+/,
+      /:\s*/,
+      /$/
+    ],
+    className: {
+      2: "attribute",
+      3: "punctuation"
+    }
+  };
+  const DICTIONARY_ITEM = {
+    match: [
+      /^\s*(?=\S)/, // have to look forward here to avoid polynomial backtracking
+      /[^:]*[^: ]/,
+      /[ ]*:/,
+      /[ ]/,
+      /.*$/
+    ],
+    className: {
+      2: "attribute",
+      3: "punctuation",
+      5: "string"
+    }
+  };
+  const STRING = {
+    match: [
+      /^\s*/,
+      />/,
+      /[ ]/,
+      /.*$/
+    ],
+    className: {
+      2: "punctuation",
+      4: "string"
+    }
+  };
+  const LIST_ITEM = {
+    variants: [
+      {
+        match: [
+          /^\s*/,
+          /-/,
+          /[ ]/,
+          /.*$/
+        ]
+      },
+      {
+        match: [
+          /^\s*/,
+          /-$/
+        ]
+      }
```

Comment:
```
list_of_dictionaries:
  -
    hello: world
    key: value

list_of_lists:
  -
    - green
    - blue
    - red
```

Would you mind adding this snippet to the markup text? This pattern wasn't immediately clear to me when reading through the NT spec. I can't help but feel this could be overlooked if not in our tests.

Reply:
Done.